Sources/Bridges/OTelSwiftLog/LogHandler.swift

```diff
@@ -0,0 +1,188 @@
+import Foundation
+import OpenTelemetryApi
+import OpenTelemetrySdk
+import Logging
+
+// let the bridgename be the url of the package?
+let bridgename: String = "OTelSwiftLog"
+let version: String = "1.0.0"
+
+public struct InstrumentationScope{
+    public var name: String
+    public var version: String?
+    public var eventDomain: String?
+    public var schemaUrl: String?
+    public var includeTraceContext: Bool?
+    public var attributes: [String: AttributeValue]?
+}
+
+// Define a custom log handler
+struct OTelLogHandler: LogHandler {
+    private var scope: InstrumentationScope // Property to store instrumentation scope name
+    private var loggerProvider : LoggerProvider  // Property to set LoggerProvider
+    private var logger: OpenTelemetryApi.Logger 
+
+    // Define the log level for this handler
```

Comment:
Is this the default level? can it be changed temporary for a specific log? These things should be part of the commentary

Reply:
Yes the default level is .info. It can be changed like this 

```
var logger = Logger(label: "com.example.myapp")
logger.logLevel = .info  // the level of of the handler
```

Reply:
Add this information to the commentary of the parameter